.github/workflows/toolkit.yml

```diff
@@ -0,0 +1,36 @@
+name: toolkit
+
+on:
+  push:
+    branches: [ "3.x" ]
+  pull_request:
+    branches: [ "3.x" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19
+    - name: Install make
+      run:  sudo apt-get update && sudo apt-get -y install make
+    - name: Build
+      run: cd src/go; make linux-amd64; cd ../../
+      
+    - name: Build the Docker image
+      run: echo "FROM oraclelinux:9-slim" > Dockerfile; echo "COPY bin/* /usr/bin/" >> Dockerfile; docker build . --file Dockerfile --tag percona-toolkit:$(date +%s)
```

Comment:
we can use `${{ github.sha }}` rather than `$(date +%s)` as tag. example: https://github.com/percona/percona-server-mongodb-operator/blob/main/.github/workflows/scan.yml#L12

Reply:
done